core/light-client/src/lib.rs

```diff
@@ -102,6 +103,12 @@ where
 		extrinsic: OpaqueExtrinsic,
 	) -> Result<(), Error>;
 
+	fn send_extrinsics<OCallApi: EnclaveOnChainOCallApi>(
+		&mut self,
+		ocall_api: &OCallApi,
+		extrinsics: Vec<OpaqueExtrinsic>,
+	) -> Result<(), Error>;
```

Comment:
The light client is extended to have the `send_extrinsics` method. The `OCallApi` is passed as parameter, rather than having it as a member of the light client. The reason being, that the light client has to implement `Encode` and `Decode`, which will be more tricky with an `Arc<OCallApi>`.

Reply:
Could you add some description here? Like that it sends the encoded extrinsics to the parentchain?

Reply:
yes, done 👍

Reply:
I think this is a good design. 👍